File: algorithm_data_structures/space_time_complexity/print_sum_and_product.js
function sumProduct(array){
	var sum = 0;
	var product = 1;
	for(var i = 0; i < array.length; i++){
		sum += array[i];
	}
	for(var j = 0; j < array.length; j++){
		product *= array[j];
	}

	console.log( sum + ", " + product);	
}


arr = [1, 2 ,3];

sumProduct( arr );

File: algorithm_data_structures/space_time_complexity/print_sum_and_product.rb
def sumProduct( arr ) 
	sum = 0
	product = 1
	for i in 0...arr.size do
		sum += arr[i]
	end
	for j in 0...arr.size do
		product *= arr[j]
	end
	
	puts "#{sum}, #{product}"

end

arr = [1,2,3]
sumProduct( arr)
